client/post-editor/edit-post-status/index.jsx

```diff
@@ -54,6 +61,7 @@ var EditPostStatus = React.createClass( {
 		stats.recordEvent( 'Changed Sticky Setting', stickyEventLabel );
 
 		actions.edit( { sticky: ! this.props.post.sticky } );
+		this.props.toggleStickyStatus( this.props.post.sticky );
```

Comment:
I purposefully copied the `toggleStickyStatus `, I think the whole logic of upper-scope `toggleStickyStatus ` function should be moved to action.
I'm even considering moving the logic to `actions.js` right now.
@mtias , what do you thing